config/chat.php

```diff
@@ -1,13 +1,12 @@
 <?php
 
 use App\Models\User;
-use App\Models\Media;
 use Illuminate\Notifications\Channels\BroadcastChannel;
 
 return [
     'classes' => [
         'user_model' => User::class,
-        'media_model' => Media::class,
+        'media_model' => RonasIT\Media\Models\Media::class,
```

Comment:
please use name without namespace

```suggestion
        'media_model' => Media::class,
```